src/Text/Pandoc/Readers/HTML.hs

```diff
@@ -588,8 +588,11 @@ pFigure = try $ do
   skipMany pBlank
   let pImg  = (\x -> (Just x, Nothing)) <$>
                (pOptInTag "p" pImage <* skipMany pBlank)
-      pCapt = (\x -> (Nothing, Just x)) <$>
-               (pInTags "figcaption" inline <* skipMany pBlank)
+      pCapt = (\x -> (Nothing, Just x)) <$> do
```

Comment:
You probably want to put a try around this one.
Otherwise it may fail after consuming some blanks, if there's no figcaption.

Reply:
`try (skipMany pBlank *> pInTags "figcaption" inline <* skipMany pBlank)`

Reply:
Actually, do you need the `skipMany pBlank`?  Won't this be handled by `pSkip`?